test/media/drm_engine_unit.js

```diff
@@ -421,6 +422,7 @@ describe('DrmEngine', () => {
             distinctiveIdentifier: 'optional',
             persistentState: 'optional',
             sessionTypes: ['temporary'],
+            initDataTypes: ['cenc'],
```

Comment:
I'm late to the party :)
May I know why you added initDataTypes in the unit tests? I saw the changes in lib were for sessionTypes. Thank you!

Reply:
@michellezhuogg 👋 
I took the PR in #2750 and strip everything related to recommendation, and the initDataTypes I added firstly in that PR, were added in https://github.com/google/shaka-player/pull/3021/files#diff-003386d961d9c0958312b9b16780a8eb91f8c57ecb8e04810a950af8819db4a2R752

I decided to keep those tests since they were developed and the PR didn't add some 😄

Reply:
Got you. Thanks!